### PR TITLE
Add ability to run read queries on the database from support

### PR DESCRIPTION
**Pull request by tijmenb**
## Context

Analysists sometimes need data from the database for analysis that's not easy to get via one of the extracts.

## Changes proposed in this pull request

Add the [Blazer gem](https://github.com/ankane/blazer/). 

It's secured via the same mechanism as Sidekiq web (so we don't have to trust Blazer's auth).

Blazer will wrap everything in a transaction, so you can't easily make UPDATE/DELETE queries on the database. But we're still looking to use [a read-only user](https://github.com/ankane/blazer/#postgresql) or read-replica for this.

## Guidance to review

I had to turn on Sprockets for this to work, is that going to cause problems?

## Link to Trello card

N/A

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
 require 'view_component/engine'
 
+require 'sprockets/railtie'
```
Comment (duncanjbrown) [inline config/application.rb:14]: CSS and JS still working on the review app. No notices on app boot. Nothing in our `assets` folder anyway. I think this is probably fine. 🤷

File: config/blazer.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
```
Comment (duncanjbrown) [inline config/blazer.yml:3]: Something in `.env.example` for this?

File: config/routes.rb [hunk 3]
```diff
@@ -808,6 +808,9 @@
     mount Sidekiq::Web => '/sidekiq', constraints: SupportUserConstraint.new
     get '/sidekiq', to: redirect('/support/sign-in'), status: 302
 
+    mount Blazer::Engine => '/blazer', constraints: SupportUserConstraint.new
```
Comment (duncanjbrown) [inline config/routes.rb:811]: 👌

**Review by duncanjbrown — APPROVED** [review]
Should be interesting! Nit about `.env`

File: config/blazer.yml (hunk 2 above)
Comment (tijmenb) [reply config/blazer.yml]: Fixed it so that it'll work in development out of the box.

**Comment by vigneshmsft** [comment]
Have merged #3581 which sets the `BLAZER_DATABASE_URL`

**Comment by duncanjbrown** [comment]
Rebased @tijmenb

**Comment by duncanjbrown** [comment]
It's good! It works!

Do we want to link from anywhere in Support?

**Pull request merged**